### DISCUSSION TITLE
Welcome to flutter_easyloading Discussions!

### DISCUSSION THREAD

**Discussion by nslogx** [nslogx/flutter_easyloading#72] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by yezhonggang** [comment]
..loadingStyle = EasyLoadingStyle.custom
      ..backgroundColor = Colors.transparent 设置透明度的时候 指示器后面有个黑背景无法去掉

当设置..backgroundColor = Colors.white 指示器会有黑色边框

**Comment by yezhonggang** [comment]
..loadingStyle = EasyLoadingStyle.custom
      ..backgroundColor = Colors.transparent 设置透明度的时候 指示器后面有个黑背景无法去掉

当设置..backgroundColor = Colors.white 指示器会有黑色边框

**Comment by yezhonggang** [comment]
..loadingStyle = EasyLoadingStyle.custom
      ..backgroundColor = Colors.transparent 设置透明度的时候 指示器后面有个黑背景无法去掉

当设置..backgroundColor = Colors.white 指示器会有黑色边框

**Comment by sumuyang168** [comment]
在MaterialApp里通过builder: EasyLoading.init()初始化时，设置app全局背景图失效，展示默认白色背景。请问如何去掉白色背景，或者改成透明？或者是否可以把FlutterEasyLoading中的_overlayEntry暴露出来？
